File: MarchFire/html2word.js
var officegen = require('officegen');
var fs = require('fs');

var docx = officegen('docx');

var pObj = docx.createP();

pObj.addText('Hello World');
pObj.addText('This is a paragraph.', { breakLine: true });

var out = fs.createWriteStream('test.docx');

out.on('error', function(err) {
  console.log(err);
});

docx.generate(out);

File: MarchFire/html2word.py
from py_mini_racer import py_mini_racer

js_code = """
var fs = require('fs');
var htmlDocx = require('html-docx-js');

var html = '<h1>Hello World</h1><p>This is a paragraph.</p>';  // Replace this with your HTML content
var docx = htmlDocx.asBlob(html);

fs.writeFileSync('test.docx', docx);
"""

ctx = py_mini_racer.MiniRacer()
ctx.eval(js_code)